src/dynarec/la64/dynarec_la64_f30f.c

```diff
@@ -230,7 +230,6 @@ uintptr_t dynarec64_F30F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
             INST_NAME("MOVQ Gx, Ex");
             nextop = F8;
             GETGX_empty(v0);
-            VXOR_V(v0, v0, v0);
             if (MODREG) {
                 v1 = sse_get_reg(dyn, ninst, x1, (nextop & 7) + (rex.b << 3), 0);
```

Comment:
This should not work in the case EX==GX, because GX is fetched as "empty" first, so EX (that is GX), will not be fetched becasue it's after the GETGX_empty(). It only work because GX has already been fetched in a previous opcode and so the cache is present (and GETGX_empty does nothing, just return the cached registred

Reply:
Ah yes, so the `GETGX_empty` need to be put after `sse_get_reg`.

Reply:
Fixed.